Dockerfile

```diff
@@ -108,13 +108,18 @@ ARG ROS_DEPS="apt-transport-https \
         libopenblas-dev \
         libpcap-dev \
         libpugixml-dev \
+        lttng-tools \
+        lttng-modules-dkms \
+        liblttng-ust-dev \
         mesa-utils \
         nano \
         nodejs \
+        python3-babeltrace \
         python3-catkin-pkg \
         python3-catkin-tools \
         python3-colcon-common-extensions \
         python3-future \
+        python3-lttng \
```

Comment:
For some reason this PR looks like it is adding new packages to be installed in the Dockerfile under ROS_DEPS, but they are already included on the `develop` branch. Would you mind looking into resolving this, or alternatively starting clean and putting up a new PR?